terraform-tests/redis_test.go

```diff
@@ -56,7 +56,7 @@ var _ = Describe("Redis", Label("redis-terraform"), Ordered, func() {
 
 		It("should create a aws_elasticache_replication_group with the right values", func() {
 			Expect(AfterValuesForType(plan, "aws_elasticache_replication_group")).To(
-				MatchKeys(IgnoreExtras, Keys{
+				MatchKeys(AllowDuplicates, Keys{
```

Comment:
Hey @pivotal-marcela-campo, `AllowDuplicates` is not strictly needed. This change was to get rid of `IgnoreExtras`.
[The available options](https://github.com/onsi/gomega/blob/master/gstruct/types.go) are `IgnoreExtras`, `IgnoreMissing` and `AllowDuplicates`.

- If we use `IgnoreExtras` the list of values is not required to be complete. So if any new property is exposed by a new version of the provider we don't get a test failure.
- If we use `IgnoreMissing` and a new version of the provider removes one of the properties the test will still pass.
- If we use `AllowDuplicates` none of the previous behaviours apply (I don't understand the specifics of `AllowDuplicates` but at least we are forced to enumerate all terraform properties and if the provider exposes new properties in the future the test will fail, and if the provider removes some of the properties it will fail too).

There is a comment in the code trying to explain this:
https://github.com/cloudfoundry/csb-brokerpak-aws/pull/755/files/a36c778aed2966a4d8354b41382bef84ecea2feb#diff-7eadf09a3d299f079d941a9f1c39e270ed2b778f0eaccf7efabbc46bf3333ac9R72-R75

Reply:
Maybe we can discuss this internally. The disadvantage is that our tests would fail for properties that we are not using and that there are no specific requests to add, so we would just modify the tests to add the required variable to prevent it from failing.

Reply:
Hopefully solved by https://github.com/cloudfoundry/csb-brokerpak-aws/pull/755/commits/392c00398e8d326dd1c4ef8685d7da2ddf2bf3c2.

I already added this as a topic for internal discussion.